src/code/audio_effects.c

```diff
@@ -160,13 +160,11 @@ f32 Audio_GetVibratoFreqScale(VibratoState* vib) {
     extent = temp + 1.0f;
     invExtent = 1.0f / extent;
 
-    // fakematch: 2^16 and 1 need to be set at the very top of this function,
-    // or else the addresses of D_80130510 and D_80130514 get computed once
-    // instead of twice. 'temp' is also a fakematch sign; removing it causes
-    // regalloc differences and reorderings at the top of the function.
-    result = 1.0f / ((extent - invExtent) * pitchChange / twoToThe16th + invExtent);
+    result = 1.0f / ((extent - invExtent) * pitchChange / 65536.0f + invExtent);
+
     D_80130510 += result;
-    D_80130514 += one;
+    D_80130514++;
+
```

Comment:
Were you able to fix this just because of the in-function static data? Nice.

Reply:
Could also probably switch `65536.0f` -> `0x10000`, but that could be done later

Reply:
yea i imagine there are a bunch of these that will get picked up in a more dedicated pass